opentreemap/treemap/tests/test_auth.py

```diff
@@ -54,6 +55,16 @@ def test_password_incorrect(self):
     #     res = self.client.get('/accounts/profile/')
     #     self.assertRedirects(res, '/accounts/profile/login/')
 
+    @override_settings(SOCIAL_AUTH_GOOGLE_OAUTH2_KEY='1', SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET='2')
+    @mock.patch('social_core.backends.oauth.OAuthAuth.state_token', return_value='zz')
+    def test_google_auth_redirect(self, *args):
```

Comment:
Actually this is not the happy case, but just redirection. I mean that this view should produce only redirection. But yes, there should be more tests, but authorization api is quite hard to test.